lib/user/getPlayerBadges.js

```diff
@@ -12,54 +12,17 @@ exports.optional = ['limit', 'cursor', 'sortOrder']
  * @alias getPlayerBadges
  * @param {number} userId - The id of the user whose badges are being fetched.
  * @param {Limit=} [limit=10] - The amount of badges being returned each request.

```

Comment:
this is actually now just a number because getPageResults can return an arbitrary number of results